src/js.cookie.js

```diff
@@ -127,7 +127,7 @@
 
 		api.set = api;
 		api.get = function (key) {
-			return api(key);
+			return api.call(api, key);
```

Comment:
It does not expects `window`. See https://github.com/js-cookie/js-cookie/issues/251#issuecomment-238235648.